src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/OperatorOverloadSyntaxClassifier.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Threading;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Classification.Classifiers;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp.Classification
+{
+    internal class OperatorOverloadSyntaxClassifier : AbstractSyntaxClassifier
```

Comment:
> AbstractSyntaxClassifier 

❓ Does this run during the syntactic classification pass or the semantic classification pass?

Reply:
It runs during semantic classification. I believe `AbstractSyntaxClassification` doesn't mean that it is syntactic classification, but that it classifies `SyntaxNodes`